.github/CONTRIBUTING.md

```diff
@@ -19,22 +17,24 @@ Once you are in the project directory, run the following commands:
 - `npm install` to pull all dependencies. Make sure to check notes below if you prefer Yarn.
 - `npm run build` to create a production version of the site.
 - `npm start` to develop on a local webpack-dev-server: [localhost:3000][3].
+
+> NOTE: run `npm fetch` before running `npm start` command for first time
+
 - `npm run fetch` to retrieve external documentation/data.
 
 The final command, `npm run fetch`, is optional as both `npm run build` and `npm start`
 will do this automatically. This step pulls in documentation for loaders/plugins hosted
 in separate repositories such as the ones found in the [webpack-contrib][4] organization.
 See the `package.json` for the full list of `scripts`.
 
-> Note that __Node 6.9.x__ or above is required for the build process to run properly.
+> Note that **Node 6.9.x** or above is required for the build process to run properly.
```

Comment:
Across the docs we use `__` vs `**` so please revert.

Reply:
Same prettier thing. 
I did the fix command with markdown lint and but still those lint staged checks where failing so I had to use no verify git flag. 
May be because of that it took the prettier changes. 
I will revert it soon